### PR TITLE
Add blog post on 2021 matching campaign

### PR DESCRIPTION
**Pull request by ericholscher**
This will go out with the announcement at the conference.

Proposed Tweet:

```
At our Portland conference, we've just announced a $1,500 donation matching campaign supporting BIPOC groups. We are suggesting a few local to Oregon, but also encourage you to find groups doing good work in your region to suport. #writethedocs 

<link to blog post> 
```

### CODE REVIEW

File: docs/blog/2021-matching-campaign.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.. post:: Apr 26, 2021
+   :tags: bipoc, statement
+
+Announcing our 2021 donation matching campaign
+==============================================
+
+We're proud to announce the second round of our $1,500 matching campaign to support social justice in our communities. `Last year <https://www.writethedocs.org/blog/we-stand-with-the-black-community/>`_ we directed donations to the Black community, and this year we're expanding that aim to focus on the wider BIPOC community. We want it to be clear that we support and care about the members of our community, and to show this through our words and our actions as an organization.
```
Comment (plaindocs) [inline docs/blog/2021-matching-campaign.rst]: We say "community" a lot.

File: docs/blog/2021-matching-campaign.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+We want to continue to make a difference in our local communities, and also encourage members of the Write the Docs community to engage with local organizations in their region. Based on our Portland conference, we're suggesting three Oregon charities we believe are doing important work. That said, you can also donate to a charity local to you doing similar outreach in your community.
```
Comment (plaindocs) [inline docs/blog/2021-matching-campaign.rst]:
```suggestion
We want to continue to make a difference in our local communities, and also encourage members of the Write the Docs community to engage with local organizations in their region. Based on our Portland conference, we're suggesting three Oregon charities we believe are doing important work. That said, you can also donate to a charity local to you doing similar outreach.
```

**Review by mxsasha — APPROVED** [review]
Should we include a mention of the unconference session? I am neutral. Other than that, looks good !

File: docs/blog/2021-matching-campaign.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+* `Asian Pacific American Network of Oregon <https://www.apano.org/>`_ - A group that unites Asian and Pacific Islanders to achieve social justice
+
+We will match donations to these organizations and others that are supporting these efforts, up to a total of $1,500. Please `forward receipts <mailto:support@writethedocs.org>`_ to us with a request for a matching donation. In the next couple months, we will post another blog post with the results of these donations, similar to `last year <https://www.writethedocs.org/blog/donations-black-community/>`_.
+
+We'd also like to thank all those in our community who are doing the work to make BIPOC members of the community feel welcome and included. This includes a BIPOC unconference at our Portland conference, a `#bipoc` channel on our Slack network, and the `links shared <https://www.writethedocs.org/blog/newsletter-april-2021/#what-we-re-reading-listening-to-watching>`_ recently in our newsletter.
+
+We hope that we can do our small part to move toward a more just world, and we appreciate everyone in the community who works with us towards these goals.
```
Comment (plaindocs) [inline docs/blog/2021-matching-campaign.rst]:
```suggestion
We hope that these actions move us all towards a more just world, and we appreciate everyone in the community who works with us towards these goals.
```

**Review by plaindocs — APPROVED** [review]
Looks good. 

Some not super helpful comments.

**Comment by ericholscher** [comment]
> Should we include a mention of the unconference session? 

@mxsasha already there: `This includes hosting a BIPOC unconference at our Portland conference,`

**Comment by mxsasha** [comment]
> > Should we include a mention of the unconference session?
> 
> @mxsasha already there: `This includes hosting a BIPOC unconference at our Portland conference,`

I somehow missed this completely

**Comment by plaindocs** [comment]
Looks good.

**Review by eirinikos — APPROVED** [review]
Beautiful!

**Comment by ericholscher** [comment]
Shipping!

**Pull request merged**